iib/workers/tasks/build.py

```diff
@@ -1417,7 +1466,7 @@ def _adjust_csv_annotations(operator_csvs, package_name, organization):
         operator_csv.dump()
 
 
-def _add_ocp_label_to_index(ocp_version, temp_dir, dockerfile_name):
+def _add_label_to_index(label_key, label_value, temp_dir, dockerfile_name):
```

Comment:
You need to update the docstring of this function to mention `label_key` and `label_value`.